src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -29,6 +29,9 @@ namespace UiaOperationAbstraction
 {
     using unique_safearray = wil::unique_any<SAFEARRAY*, decltype(&::SafeArrayDestroy), ::SafeArrayDestroy>;
 
+    // Forward declare UiaElement as it is used in a constructor for UiaVariant 

```

Comment:
You don't have to worry about adding a comment for this, especially since the reason(s) for being forward declared could change over time. :)